Libraries/Text/RCTShadowText.m

```diff
@@ -266,9 +266,16 @@ - (NSAttributedString *)_attributedStringWithFontFamily:(NSString *)fontFamily
 
   _effectiveLetterSpacing = letterSpacing.doubleValue;
 
-  UIFont *font = [RCTConvert UIFont:nil withFamily:fontFamily
-                               size:fontSize weight:fontWeight style:fontStyle
-                    scaleMultiplier:_allowFontScaling ? _fontSizeMultiplier : 1.0];
+  NSArray *fontVariant = _fontVariant ? _fontVariant : NULL;
```

Comment:
Not sure what you're trying to do here, but you can just pass _fontVariant directly to the method below.